### ISSUE TITLE
Remove Fresco

### ISSUE THREAD

**Issue by TacoTheDank** [fenimore/DemocracyDroid#60]
Removing this library would singlehandedly drop the app size by about 5 megabytes. I tried doing it myself but the images broke and would no longer display so I'm not sure how to go about doing it.

Fresco is a MASSIVE library, and unless we absolutely need to use it, I think it would be better if it were removed.

**Comment by fenimore** [comment]
Yes I'd really like to do that, I'm not sure exactly what it's used for but it's been there for a while so maybe there are easy replacements
